lib/api/CInferenceModelDefinition.cc

```diff
@@ -227,6 +285,50 @@ void CEnsemble::classificationWeights(TDoubleVec classificationWeights) {
     this->CTrainedModel::classificationWeights(std::move(classificationWeights));
 }
 
+CTrainedModel::TSizeInfoUPtr CEnsemble::sizeInfo() const {
+    return std::make_unique<CSizeInfo>(*this);
+}
+
+CEnsemble::CSizeInfo::CSizeInfo(const CEnsemble& ensemble)
+    : CTrainedModel::CSizeInfo(ensemble), m_Ensemble{&ensemble} {
+}
+
+std::size_t CEnsemble::CSizeInfo::numOperations() const {
+    std::size_t numOperations{0};
+    for (const auto& model : m_Ensemble->m_TrainedModels) {
+        numOperations += model->sizeInfo()->numOperations();
+    }
+    return numOperations;
+}
+
+void CEnsemble::CSizeInfo::addToDocument(rapidjson::Value& parentObject,
+                                         TRapidJsonWriter& writer) const {
+    this->CTrainedModel::CSizeInfo::addToDocument(parentObject, writer);
+    rapidjson::Value featureNameLengthsArray =
+        writer.makeArray(m_Ensemble->featureNames().size());
```

Comment:
You assign to `rapidjson::Value` in quite a few places rather than using `{}`-initialisation, but there isn't a problem with this right? I'd prefer:
```suggestion
    rapidjson::Value featureNameLengthsArray{
        writer.makeArray(m_Ensemble->featureNames().size())};
```
for consistency.

Reply:
I was consistently inconsistent in this file :see_no_evil: It is fixed now.